apigrpc/apigrpc.swagger.json

```diff
@@ -1379,28 +1498,6 @@
             }
           }
         },
-        "parameters": [
```

Comment:
Can you revert this removal? We manually patch these parameters into the Swagger spec because of a bug in the protoc openapi-codegen plugin.